kv/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
Please add as many Clippy lints as possible.

Reply:
I will add in next PR.

Reply:
@rogercloud Please add for curp